policyengine_us/parameters/gov/states/mn/tax/income/credits/marriage/minimum_taxable_income.yaml

```diff
@@ -2,13 +2,18 @@ description: Minnesota provides a marriage credit for joint filers where their t
 values:
   2021-01-01: 40_000
   2022-01-01: 42_000
+  2023-01-01: 44_000
 
 metadata:
   period: year
   unit: currency-USD
   label: Minnesota marriage credit minimum taxable income for eligibility
   reference:
+  - title: Minnesota Statutes 290.0675 MARRIAGE PENALTY CREDIT. Subd. 3.Credit amount.
```

Comment:
Similar to above (and always) - if the legal code doesn't support the parameter values, please explain in a comment. Please check all parameters for this.